### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by tanya-98**
__Which issue does this PR address?__

Closes #

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ An integrated modeling solution for BPMN, DMN and CMMN based on [bpmn.io](http:/
 ## Building the Application
 
 ```sh
-# checkout a tag
+# checkout tag
```
Comment (pinussilvestrus) [inline README.md:22]: Can you please explain why this change is an improvement?

**Comment by philippfromme** [comment]
@tanya-98 Are you serious?

**Comment by tanya-98** [comment]
yes
